engine-nextgen/src/test/kotlin/graphql/nadel/validation/NadelEnumValidationTest.kt

```diff
@@ -114,7 +114,7 @@ class NadelEnumValidationTest : DescribeSpec({
             val errors = validate(fixture)
             assert(errors.isNotEmpty())
 
-            val error = errors.singleOfType<MissingUnderlyingEnumValue>()
+            val error = errors.assertSingleOfType<MissingUnderlyingEnumValue>()
```

Comment:
Ok sorry I cluttered the PR with a lot of these changes.

This new function is a lot better when it fails. It's tailored to the errors and prints out the error types or messages if it could not return a proper single.